tutorials/jmsActiveMQ/src/test/java/com/example/jmsActiveMQ/ApplicationTest.java

```diff
@@ -0,0 +1,205 @@
+/**
+ * Put your copyright and license info here.
+ */
+package com.example.jmsActiveMQ;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashSet;
+
+import javax.jms.Connection;
+import javax.jms.DeliveryMode;
+import javax.jms.Destination;
+import javax.jms.JMSException;
+import javax.jms.MessageProducer;
+import javax.jms.Session;
+import javax.jms.TextMessage;
+import javax.validation.ConstraintViolationException;
+
+import org.junit.Assert;
+import org.apache.activemq.ActiveMQConnectionFactory;
+import org.apache.activemq.broker.BrokerService;
+import org.apache.hadoop.conf.Configuration;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+import com.datatorrent.api.LocalMode;
+import com.example.jmsActiveMQ.ActiveMQApplication;
+
+/**
+ * Test the DAG declaration in local mode.
+ */
+public class ApplicationTest {
+
+  private static final Logger LOG = LoggerFactory.getLogger(ApplicationTest.class);
+  
+  private static final String FILE_NAME = "test";
+  private static final String FILE_DIR  = "/tmp/FromAMQ";
```

Comment:
Better to use relative path beginning with "target/" as done in some of the other examples (e.g. hdfs2kafka) since that is a known transient directory.